shells/pipes-shell/source/verbs/run-arc.js

```diff
@@ -61,7 +61,7 @@ const instantiateRecipe = async (arc, recipe, particleId, particleName, provided
 
   plan = updateParticleInPlan(plan, particleId, particleName, providedSlotId);
   if (!plan) {
-    warn(`failed updating particle id '${particleId}', name ${particleName} in plan ${plan.toString()}`);
+    warn(`failed updating particle id '${particleId}', name ${particleName}`);
```

Comment:
can you use `recipe` here instead of `plan`... but yeah, this is a funny one, sorry about that.

Reply:
Done!